### DISCUSSION TITLE
Possible to release new beta for KEDA v2?

### DISCUSSION THREAD

**Discussion by mboutet** [kedacore/keda#1199] [Q&A / Need Help] [answered]
Hi maintainer of KEDA, I was wondering if there was a plan to release another beta of KEDA v2 with all the fixes that have been made since the first beta? In particular, https://github.com/kedacore/keda/pull/1169 and https://github.com/kedacore/keda/pull/1172 are required to test KEDA v2 in my case since I'm using RabbitMQ queues with "illegal" characters (`/`) in their name.

Thank you very much!

**Comment by tomkerkhove** [comment, marked as answer]
We'll look into it, thanks for the suggestion!
